src/Renderer/ThreeExtended/Feature2Texture.js

```diff
@@ -71,17 +74,21 @@ function drawPoint(ctx, vertice, origin, scale, style = {}) {
 }
 
 function drawFeature(ctx, feature, origin, scale, extent, style = {}) {
-    const properties = feature.properties;
-
-    if (typeof (style) == 'function') {
-        style = style(properties, feature);
-    }
-
     for (const geometry of feature.geometry) {
+        const properties = geometry.properties;
+
+        let gStyle = style;
```

Comment:
This could be reverted outside of the loop

Reply:
that is to say? 
`properties` may change each step

Reply:
Yes indeed if `style` is a function, but you can declare `gStyle = style` outside the loop

Reply:
yes